contentcloud-spring-boot-platform/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     constraints {
         api project(':contentcloud-spring-boot-starter')
         api project(':contentcloud-spring-boot-starter-annotations')
+        api "eu.xenit.contentcloud.thunx:thunx-api-spring-boot-starter:${thunxVersion}"
```

Comment:
Please also add the rest of the thunx artifacts to the platform.

Then you will still be able to use the platform while not using the starter.